django/db/backends/oracle/creation.py

```diff
@@ -189,12 +189,12 @@ def _execute_test_db_creation(self, cursor, parameters, verbosity, keepdb=False)
             print("_create_test_db(): dbname = %s" % parameters['user'])
         statements = [
             """CREATE TABLESPACE %(tblspace)s
-               DATAFILE '%(datafile)s' SIZE 20M
-               REUSE AUTOEXTEND ON NEXT 10M MAXSIZE %(maxsize)s
+               DATAFILE '%(datafile)s' SIZE 50M
+               REUSE AUTOEXTEND ON NEXT 20M MAXSIZE %(maxsize)s
             """,
             """CREATE TEMPORARY TABLESPACE %(tblspace_temp)s
-               TEMPFILE '%(datafile_tmp)s' SIZE 20M
-               REUSE AUTOEXTEND ON NEXT 10M MAXSIZE %(maxsize_tmp)s
+               TEMPFILE '%(datafile_tmp)s' SIZE 50M
+               REUSE AUTOEXTEND ON NEXT 20M MAXSIZE %(maxsize_tmp)s
```

Comment:
If we already have `DATAFILE_MAXSIZE` and `DATAFILE_TMP_MAXSIZE` then it may be worth to add new test settings as you proposed (ticket [#28260](https://code.djangoproject.com/ticket/28260#ticket)). We can avoid tablespace datafile autoextend which can be painful in a big test suites like ours.